tdrs-backend/tdpservice/security/models.py

```diff
@@ -1,53 +1,20 @@
 """Models for the tdpservice.security app."""
-from hashlib import sha256
+import logging
 from io import StringIO
 from os.path import join
 from typing import Union
-import logging
 
-from django.contrib.admin.models import ContentType, LogEntry, ADDITION
+from django.contrib.admin.models import ADDITION, ContentType, LogEntry
 from django.core.files.base import File
 from django.db import models
 from django.utils.timezone import now
 
 from tdpservice.backends import DataFilesS3Storage
-from tdpservice.data_files.models import DataFile
+from tdpservice.data_files.models import DataFile, get_file_shasum
```

Comment:
both the data files and the security models need to create shasums, but security requies data files as a dependency, so if we imported get file shasum from here it breaks the app.